### PR TITLE
Get argocd route app UI

### PR DESCRIPTION
**Pull request by chenz4027**
image

### CODE REVIEW

**Review by fxiang1 — APPROVED** [review]
Looks good! Thanks!

File: src-web/components/common/OverviewCards/index.js [hunk 1]
```diff
@@ -279,33 +298,71 @@ class OverviewCards extends React.Component {
...
+              <AcmActionGroup>
+                <AcmButton
+                  href={this.state.argoLink}
+                  variant={ButtonVariant.link}
+                  id="launch-argocd-editor"
+                  component="a"
+                  rel="noreferrer"
+                  icon={<ExternalLinkAltIcon />}
+                  iconPosition="right"
+                >
```
Comment (KevinFCormier) [inline src-web/components/common/OverviewCards/index.js:313]: 1. Did you consider whether we want the link to open in the same tab or a new one? I think design has opinions on this.
2. Should we grey out or not display this link if for some reason we cannot get the URL for it?

**Review by KevinFCormier — COMMENTED** [review]
A couple things to look into...

File: src-web/components/common/OverviewCards/index.js (hunk 1 above)
Comment (chenz4027) [reply src-web/components/common/OverviewCards/index.js:313]: Thanks @KevinFCormier , will look into those two scenarios

**Review by KevinFCormier — APPROVED** [review]

**Pull request merged**